src/com/egs/arsensargsyan/task2/ReadFromFileIntoArrayList.java

```diff
@@ -0,0 +1,110 @@
+package task2;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ReadFromFileIntoArrayList {
+
+    private static ArrayList<User> users = new ArrayList<>();
+    private static final File FILE_PATH = new File("arsen-sargsyan-resources/fileArrayList");
```

Comment:
I would remove statics and pass the file path via constructor, because this functional could be used with other paths as well... make user decide where to write and read from